packages/provider/src/tests/runProviderTest.ts

```diff
@@ -7,1803 +7,1805 @@ export function runProviderTest<
   Options extends JoshProvider.Options = JoshProvider.Options,
   CleanupProvider extends JoshProvider = JoshProvider
 >(options: ProviderTestOptions<Provider, Options, CleanupProvider>): void {
-  const { providerConstructor: Provider, providerOptions = {}, cleanup } = options;
+  const { providerConstructor: Provider, providerOptions = { disableSerialization: false }, cleanup, disableDisableSerializationCheck } = options;
```

Comment:
- Better name for `disableDisableSerializationCheck`
- Why are you adding the option for `disableSerialization` in `providerOptions`?

Reply:
I named it that because it is accurate haha, there's not much to change since you named it disableSerialization instead of serialization: false
As for the property, I left it in by accident and I'll fix that 👍

Reply:
I think you should change the option to `serialization` and just make sure it's description states what it does.